engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/views/docker-discovery-create.html

```diff
@@ -0,0 +1,190 @@
+                    ng-model="createRepoChoices.existingProductId"
+                    ng-options="product.id as product.name for product in products"/>
+          </dd>
+        </dl>
+        <dl class="dl-horizontal dl-horizontal-left" ng-show="createRepoChoices.newProduct === 'true'">
+          <dt translate>Name</dt>
+          <dd>
+            <input ng-model="createRepoChoices.product.name"
+                   id="name"
+                   name="name"
+                   type="text"
+                   class="form-control"
+                   tabindex=1
+                   ng-required="createRepoChoices.newProduct === 'true'"/>
+          </dd>
+          <dt translate>Label</dt>
+          <dd>
+            <input ng-model="createRepoChoices.product.label"
+                   id="label"
+                   name="label"
+                   type="text"
+                   class="form-control"
+                   tabindex=2
+                   ng-required="createRepoChoices.newProduct === 'true'"/>
+          </dd>
+          <dt translate>GPG Key</dt>
+          <dd>
+            <select class="form-control" ng-model="createRepoChoices.product.gpg_key_id"
+                    ng-options="gpg_key.id as gpg_key.name for gpg_key in gpgKeys"
+                    ng-disabled="gpgKeys.length === 0"/>
+          </dd>
+        </dl>
+      </div>
+
+      <h4 translate>Repository Options</h4>
+      <div bst-form-group>
+        <dl class="dl-horizontal dl-horizontal-left">
+          <dt translate>Url</dt>
+          <dd>
+            <input type="text" class="form-control"
+                   id="url"
+                   name="url"
+                   ng-model="createRepoChoices.repository_url"
+                   ng-disabled="creating()"/>
+          </dd>
+          <dt translate>Verify SSL</dt>
+          <dd>
+            <input type="checkbox"
+                   id="verify_ssl"
+                   ng-model="createRepoChoices.verify_ssl"
+                   ng-disabled="creating()"/>
+          </dd>
+          <dt translate>Upstream Username</dt>
+          <dd>
+            <input type="text" class="form-control"
+                   id="upstream_username"
+                   name="upstream_username"
+                   ng-model="createRepoChoices.upstream_username"
+                   ng-disabled="creating()"/>
+          </dd>
+          <dt translate>Upstream Password</dt>
+          <dd>
+            <input type="text" class="form-control"
+                   id="upstream_password"
+                   name="upstream_password"
+                   ng-model="createRepoChoices.upstream_password"
+                   ng-disabled="creating()"/>
+          </dd>
+        </dl>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <h4 translate>Repositories to Create</h4>
+    <div data-extend-template="layouts/partials/table.html">
+      <div data-block="search">
+        <div class="input-group">
```

Comment:
The class (and the `<div>` itself) is unneeded here since this isn't an input group.